webpack/assets/javascripts/react_app/components/DualList/DualList.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DualListControlled as PfDualList } from 'patternfly-react';
+import { arrangeItemsBySelectedIDs } from './DualListHelper';
+
+import './DualList.scss';
+
+const DualList = props => {
+  const { id, items, selectedIDs, inputProps, disabled, error } = props;
+  const { selectedList, unselectedlist } = arrangeItemsBySelectedIDs(
```

Comment:
For making it work in react pages (no erb mounting), this component needs to fetch the items by an API request.
the flow should be like this
```js
// ConnectedDualList
const DualList = ({fetchDualItems, ...otherProps}) => {
  const dispatch = useDispatch(); // creating a redux action dispatcher  
  useEffect(() => {
    dispatch(fetchDualItems); // this will dispatch an action, creates ab API call and updates redux store
  }, [dispatch, fetchDualItems]);
  const unSelectedList = useSelector(selectAllSelectedList) // this invoke a selector so select a specific part of the 
  store, it can be memoreized. 
  const unSelectedList = useSelector(selectAllUnselectedList) 
  return <DualList selectedList={unSelectedList} unSelectedList={unSelectedList} ...otherProps />
}
```
It can be a part of a follow-up PR, and it doesn't require for mounted components
if you need some assitance feel free to ping me on irc (/afeferku)